pyvo/dal/mimetype.py

```diff
@@ -75,32 +75,37 @@ def mime_object_maker(url, mimetype, session=None):
     if not mimetype:
         raise ValueError('mimetype required')
     session = use_session(session)
-    full_type, params = cgi.parse_header(mimetype)
-    mimetype = [x.strip() for x in full_type.split('/')] if '/' in full_type \
+    m = Message()
+    m['content-type'] = mimetype
+    pp = m.get_params()
+    full_type = pp[0][0]
+    params = pp[1:]
+    mtype = [x.strip() for x in full_type.split('/')] if '/' in full_type \
         else None
```

Comment:
We allow longer lines
```suggestion
    mtype = [x.strip() for x in full_type.split('/')] if '/' in full_type else None
```